2.x/Dockerfile

```diff
@@ -71,15 +72,6 @@ RUN set -x \
         tcl \
         icu-dev \
         wget \
-    && : "---------- curl ----------" \
```

Comment:
Why only on 2.x?

Reply:
Oh, I did not notice that milestone of [this](https://github.com/tarantool/tarantool/issues/4318) issue was changed to 1.10.4

Reply:
Amended and pushed again